addons/bestja_organization/views/organization.xml

```diff
@@ -12,6 +12,11 @@
                 ('active', '=?', False),
                 ('coordinator.id', '=', uid),
             ]</field>
+            <field name="help" type="html">
+                <p class="oe_view_nocontent_create">
+                    Zostań organizatorem wolontariatu! Sprawdź w bazie wiedzy, jakie trzeba spełnić warunki, a następnie utwórz organizację
```

Comment:
All `oe_view_nocontent_create` messages (i.e. a short label for an arrow pointing to the "Create" button) don't include a dot, all other messages include a dot. So I think it is consistent :)